src/containers/App/App.js

```diff
@@ -23,6 +23,7 @@ function App() {
   useEffect(() => {
       getAllJobs()
         .then(data => {
+          console.log('fetching!')
```

Comment:
we should get rid of this

Reply:
I'm sure I've got a few more console logs hiding in there! I'll do a search for all of them to make sure all of them are removed.